examples/confluence/main.tf

```
@@ -0,0 +1,308 @@
+  vpc_id      = module.vpc.vpc_id
+}
+
+module "asg-to-world" {
+  source            = "fpco/foundation/aws//modules/open-egress-sg"
+  security_group_id = module.asg-sg.id
+}
+
+module "ssh-port-sg-rule" {
+  source            = "fpco/foundation/aws//modules/single-port-sg"
+  security_group_id = module.asg-sg.id
+  cidr_blocks       = ["0.0.0.0/0"]
+  port              = 22
+  description       = "SSH from anywhere, for debug."
+}
+
+module "asg_int_alb_http_port_sg_rule" {
+  source                   = "git::ssh://git@github.com/fpco/terraform-aws-foundation//modules/single-port-sg?ref=spsg"
+  security_group_id        = module.asg-sg.id
+  port                     = 80
+  description              = "HTTP ingress for int ALB"
+  source_security_group_id = module.int-alb.security_group_id
+}
+
+module "asg_ext_alb_http_port_sg_rule" {
+  source                   = "git::ssh://git@github.com/fpco/terraform-aws-foundation//modules/single-port-sg?ref=spsg"
+  security_group_id        = module.asg-sg.id
+  port                     = 80
+  description              = "HTTP ingress for ext ALB"
+  source_security_group_id = module.ext-alb.security_group_id
+}
+
+module "asg" {
+  source             = "git::ssh://git@github.com/fpco/terraform-aws-foundation//modules/single-node-asg?ref=lb-asg"
+  ami                = module.centos.id
+  instance_type      = "m5.xlarge"
+  key_name           = var.ssh_key
+  name_prefix        = "confluence"
+  name_suffix        = ""
+  region             = var.region
+  security_group_ids = [module.asg-sg.id]
+  subnet_id          = module.vpc.private_subnet_ids[0]
+  public_ip          = false
+  data_volume_size   = 50
+  init_prefix        = <<EOF
+yum install -y python3-pip
+pip3 install awscli
+${module.install-docker-compose.init_snippet}
+EOF
+  init_suffix        = <<EOF
+mkdir -p /data
+mkfs.xfs /dev/xvdf
+mount /dev/xvdf /data
+mkdir -p /data/confluence
+cat > /tmp/docker-compose.yml <<EOCAT
+${data.template_file.docker_compose.rendered}
+EOCAT
+cd /tmp
+docker-compose up -d
+# rm docker-compose.yml
+EOF
+}
+
+data "template_file" "docker_compose" {
+  template = file("${path.module}/docker-compose.tpl")
+  vars = {
+    http_port = 80
+    db_host   = module.rds.endpoint
+    db_db     = "confluence"
+    db_user   = "confluence"
+    db_pass   = var.db_password
+  }
+}
+
+module "data-backup" {
+  source          = "git::ssh://git@github.com/fpco/terraform-aws-foundation//modules/dlm-lifecycle-policy?ref=dlm"
+  name_prefix     = "confluence"
+  ebs_target_tags = { Name = module.asg.data_volume_name_tag }
+}
+
+module "install-docker-compose" {
+  source = "git::ssh://git@github.com/fpco/terraform-aws-foundation//modules/init-snippet-install-docker-yum?ref=install-docker"
+}
+
+module "rds-sg" {
+  source      = "fpco/foundation/aws//modules/security-group-base"
+  name        = "rds-sg"
+  description = "SG for RDS"
+  vpc_id      = module.vpc.vpc_id
+}
+
+module "rds_sg_rule" {
```

This is a really small thing but we typically want to use kebab case for module names and snakecase for resources. 
```suggestion
module "rds-sg-rule" {
```